sympy/combinatorics/testutil.py

```diff
@@ -192,15 +201,20 @@ def _verify_normal_closure(group, arg, closure=None):
 
 def canonicalize_naive(g, dummies, sym, *v):
     """
-    Canonicalize tensor formed by tensors of the different types
+    Canonicalize tensor formed by tensors of the different types.
 
-    g  permutation representing the tensor
-    dummies  list of dummy indices
-    msym symmetry of the metric
+    Parameters
+    ==========
 
-    v is a list of (base_i, gens_i, n_i, sym_i) for tensors of type `i`
-    base_i, gens_i BSGS for tensors of this type
-    n_i  number ot tensors of type `i`
+    g : Permutation representing the tensor.
+    dummies : List of dummy indices.
+    msym : Symmetry of the metric.
+    v : A list of (base_i, gens_i, n_i, sym_i) for tensors of type `i`.
+        base_i, gens_i BSGS for tensors of this type
+        n_i  number ot tensors of type `i`
+
+    Explanation
```

Comment:
Can you move the explanation before the parameters?

Reply:
Done.